transport/local-transport/src/main/java/org/apache/airavata/mft/transport/local/LocalIncomingChunkedConnector.java

```diff
@@ -32,13 +32,15 @@ public class LocalIncomingChunkedConnector implements IncomingChunkedConnector {
 
     private String resourcePath;
     private long resourceSize;
+    private boolean dmaFlag;
 
     private static final Logger logger = LoggerFactory.getLogger(LocalIncomingChunkedConnector.class);
 
     @Override
     public void init(ConnectorConfig connectorConfig) throws Exception {
         this.resourcePath = connectorConfig.getResourcePath();
         this.resourceSize = connectorConfig.getMetadata().getFile().getResourceSize();
+        this.dmaFlag = connectorConfig.getTransportConfig().get("local.dma").toString().equals("true");
```

Comment:
It is better if there is a utility class to parse these configs. It will enable proper error handling and automatic type parsing

Booelan dma = TransportProperties.getBoolean(connectorConfig.getTransportConfig(), "local.dma", True) // Here we parse default value if the property is not available in the transport config